src/attr/_funcs.py

```diff
@@ -216,6 +217,9 @@ def evolve(inst, **changes):
     try:
         return cls(**changes)
     except TypeError as exc:
-        k = exc.args[0].split("'")[1]
-        raise AttrsAttributeNotFoundError(
-            "{k} is not an attrs attribute on {cl}.".format(k=k, cl=cls))
+        for name in changes:
+            if getattr(attrs, name, None) is None:
+                k = exc.args[0].split("'")[1]
```

Comment:
I don't think we need this any more. We have the name of the attribute already in `name`.